test_scripts/Resumption/Handling_errors_from_HMI/007_resume_all_data_with_one_rpc_2apps_unexpected_disconnect.lua

```diff
@@ -0,0 +1,87 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
rename test to 007_resume_all_data_with_one_error_rpc_2apps_unexpected_disconnect.lua